yellowbrick/features/base.py

```diff
@@ -124,6 +130,12 @@ def fit(self, X, y=None, **fit_params):
 ## Data Visualizers
 ##########################################################################
 
+class target_type(Enum):
+    AUTO = 'auto'
+    SINGLE = 'single'
+    DISCRETE = 'discrete'
+    CONTINUOUS = 'continuous'
```

Comment:
So do you like the `Enum`? What happens when you do `target_type("foo")`" From the below code it appears that some exception is raised; do you know what kind? 

Also, this should probably be `class TargetType(enum)` - so that we can do `TargetType.AUTO`.

Reply:
Yup, the enums are a great way to refer to a group of constants having the same functionality. `target_type("foo")` raises `ValueError` stating that `"foo"` is not a part of the class.

I will change the name to TargetType.